app/src/main/java/com/keylesspalace/tusky/MainActivity.kt

```diff
@@ -257,6 +261,23 @@ class MainActivity : BottomSheetActivity(), ActionButtonActivity, HasAndroidInje
         val hideTopToolbar = preferences.getBoolean(PrefKeys.HIDE_TOP_TOOLBAR, false)
         binding.mainToolbar.visible(!hideTopToolbar)
 
+        // Configure a faux-options menu if the top toolbar is hidden
+        if (hideTopToolbar) {
+            val navOnBottom = preferences.getString(PrefKeys.MAIN_NAV_POSITION, "top") == "bottom"
+            val view = if (navOnBottom) binding.bottomNavOptionsMenu else binding.topNavOptionsMenu
+            val popup = PopupMenu(this, view)
```

Comment:
There is no "system" way of doing this?

(When the title is hidden there is no context menu?)

This fixes then another issue: If at the moment the title is hidden, "Refresh" and "Search" are not accessible.

It would be quite a bit deal here then.
I would move this code to a method then, something like `if (hideTopToolbar) emulateContextMenuForNavBar()`.